### PR TITLE
update vsphere ci test frequency

### PR DESCRIPTION
**Pull request by jianlinliu**
1. we often hit vsphere cup quota limit, adjust some vsphere job's test frequency to f14 from f7.
2. change some job's f21 to f28

### CODE REVIEW

**Comment by jianlinliu** [comment]
cc @liangxia @jhou1 @LiangquanLi930 @WenXinWei @sgaoshang to review.

**Comment by LiangquanLi930** [comment]
Thanks a lot
/lgtm

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.12.yaml [hunk 1]
```diff
@@ -65,7 +65,7 @@ resources:
       memory: 200Mi
 tests:
 - as: aws-c2s-ipi-disconnected-private-p2-f14
-  cron: 34 11 5,20 * *
+  cron: 28 23 8,23 * *
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.12.yaml:68]: @jianlinliu Any update to the script ?
This  is what i got,
```
[lxia@homepc release]$ ./ci-operator/config/openshift/openshift-tests-private/tools/generate-cron-entry.sh aws-c2s-ipi-disconnected-private-p2-f14 openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.12.yaml
cron: 34 11 5,20 * *
```

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.12.yaml:68]: Interesting, you are right, let me update it again.

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.12.yaml]: Okay, I found the root cause, I gave a full path for yaml file, so generated different cron entries.

Now they are fixed, pls help review.

**Pull request merged**